dist/index.html

```diff
@@ -3,19 +3,16 @@
 <head>
     <meta charset="UTF-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
-    <script crossorigin src="https://unpkg.com/react@16/umd/react.production.min.js"></script>
-    <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.production.min.js"></script>
-    <script type="application/javascript" src="./bundle.js"></script>
     <title>Onboarding Web Page</title>
 </head>
 <body>
-    <div id="header">
+    <header>
         <h1>Lab for Josephine</h1>
-    </div>
-    <button id="apiButton" type="button" >Get Twitter Timeline</button>
-    <div id="apiDataContainer">
-        <p>Loading your Twitter timeline...</p>
-    </div>
-    <div id="reactContainer"></div>
+    </header>
+    <div id="root"></div>
+
+    <script crossorigin src="https://unpkg.com/react@16/umd/react.production.min.js"></script>
```

Comment:
why is this here?

Reply:
best practice is to put script tags before the closing body tags. That way the browser loads the DOM before loading the scripts. Otherwise conflicts occur when the scripts act on DOM elements that don't exist yet